lib/gui/lvgl_common/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: Apache-2.0
+
+
+set(ZEPHYR_CURRENT_LIBRARY lvgl)
```

Comment:
I can't see how this line of code is affecting anything, which piece of code is reading ZEPHYR_CURRENT_LIBRARY and getting lvgl because of this line?

Reply:
@SebastianBoe it was not needed